source/help/getting-started/searching.rst

```diff
@@ -1,101 +1,147 @@
--  You can use search modifiers such as ``from:dave`` to return results only from certain people or in certain channels. For more information about this, see the `Search Modifiers`_ section.
+.. tip::
+  You can now easily access files recently shared in any channel by selecting the **Channel Files** icon below the channel name. 
+  
+  .. image:: ../../images/channel-files-icon.png
+
+All Searches
+------------
+
+- Multiple-word searches return results that contain *all* of your search criteria.
+- Search modifiers can help narrow down searches. See the `Search Modifiers`_ section below for details.
+- You can search Archived channels as long as you're a member of that channel. 
+
+  - If you're unable to see messages in archived channels in your search results, ask your System Admin if **Allow users to view archived channels** has been disabled under **System Console > Site Configuration > Users and Teams**. 
+  - To remove archived channels from your search results, you can leave Archived channels. 
+- Like many search engines, common words such as ``the``, ``which``, and ``are`` (known as "stop words"), as well as two-letter and one-letter search terms, are not shown in search results because they typically return too many results. See the `Technical Notes`_ section below for details.
+- IP addresses (e.g. ``10.100.200.101``) don't return results.
+- When no search type is selected, message search results are returned first.
+
+.. Tip::
+  In the **Search Results** pane, you can switch between message and file search results by selecting the **Messages** and **Files** options.
+  
+  .. image:: ../../images/review-all-search-results.png
+
+Searching for Messages 
+----------------------
+
+Select the search option, then select **Messages** as the search type, followed by your search criteria. 
+
+.. image:: ../../images/search-messages.png
+
+When message results display in the **Search Results** pane, select **Jump** to view the full message in context.
+
+.. image:: ../../images/jump-to-message.png
+
+Searching for Files
+-------------------
+
+Select the search option, then select **Files** as the search type, followed by your search criteria. Search returns file attachments that match on file name or contain matching text content within supported document types. Each search result includes the file name, extension, size, as well as when and where the file was originally attached.
+
+.. image:: ../../images/search-files.png
 
-Like many search engines, common words such as ``the``, ``which``, and ``are`` (known as "stop words"), as well as two-letter and one-letter search terms, are not shown in search because they typically return too many results. See database documentation on `MySQL`_ and `Postgres`_ for a full list.
+- For Mattermost Cloud workspaces, supported document file formats include PDF, DOCX, PPTX, ODT, HTML, and plain text documents. 
```

Comment:
@esethna @jespino  - Can Cloud end users search on RTF and PAGES file formats, as well as the contents of ZIP files as of April 21? What about file attachments shared prior to file search being available?